src/main/java/org/gridsuite/voltageinit/server/service/parameters/VoltageInitParametersService.java

```diff
@@ -91,35 +99,51 @@ private Map<String, VoltageLimitEntity> resolveVoltageLevelLimits(VoltageInitRun
         return voltageLevelLimits;
     }
 
-    private void fillSpecificVoltageLimits(List<VoltageLimitOverride> specificVoltageLimits, Map<String, VoltageLimitEntity> voltageLevelModificationLimits, Map<String, VoltageLimitEntity> voltageLevelDefaultLimits, VoltageLevel voltageLevel, Map<String, Double> voltageLevelsIdsRestricted) {
+    @SuppressWarnings({"EnumSwitchStatementWhichMissesCases", "java:S131"})
+    // it's wanted to not have a case for NONE in switch, as we do nothing in this case
```

Comment:
Then why do we even need the NONE case?
If it's necessary somehow, add the missing statement in the switch with the comment, don't suppress this warning as it could hide a real mistake.

Reply:
forget to remove it, no needed anymore with ifs